pwndbg/dbg/lldb/repl/readline.py

```diff
@@ -9,11 +9,19 @@
 import contextlib
 import functools
 import os.path
+import sys
 from typing import Callable
 from typing import ParamSpec
 from typing import TypeVar
 
-import gnureadline as readline
+if sys.platform == "win32":
+    import readline
+
+    # pyreadline3 doesn't implement `set_completion_display_matches_hook`
+    readline.set_completion_display_matches_hook = lambda *args: None
```

Comment:
Shall we make a check for that with some warning in case windows implement that one day?

Reply:
fixed